packages/common/src/compiler/filtersCompiler.test.ts

```diff
@@ -806,3 +812,197 @@ describe('Filter SQL', () => {
+            {
+                target: { fieldRef: 'field2' },
+                id: '1',
+                operator: FilterOperator.EQUALS,
+                values: [false],
+            },
+            {
+                target: { fieldRef: 'field3' },
+                id: '1',
+                operator: FilterOperator.NOT_EQUALS,
+                values: [true],
+            },
+        ];
+        const expected: DbtColumnLightdashMetric['filters'] = [
+            { field2: 'false' },
+            { field3: '!true' },
+        ];
+        expect(convertMetricFilterToDbt(filters)).toEqual(expected);
+    });
+
+    it('should convert INCLUDE filters correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'field1' },
+                id: '1',
+                operator: FilterOperator.INCLUDE,
+                values: ['katie'],
+            },
+        ];
+        const expected: DbtColumnLightdashMetric['filters'] = [
+            { field1: '%katie%' },
+        ];
+        expect(convertMetricFilterToDbt(filters)).toEqual(expected);
+    });
+
+    it('should convert STARTS_WITH and ENDS_WITH filters correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'field1' },
+                id: '1',
+                operator: FilterOperator.STARTS_WITH,
+                values: ['katie'],
+            },
+            {
+                target: { fieldRef: 'field2' },
+                id: '2',
+                operator: FilterOperator.ENDS_WITH,
+                values: ['katie'],
+            },
+        ];
+        const expected: DbtColumnLightdashMetric['filters'] = [
+            { field1: 'katie%' },
+            { field2: '%katie' },
+        ];
+        expect(convertMetricFilterToDbt(filters)).toEqual(expected);
+    });
+
+    it('should convert GREATER_THAN and GREATER_THAN_OR_EQUAL filters correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'field1' },
+                id: '1',
+                operator: FilterOperator.GREATER_THAN,
+                values: ['4'],
+            },
+            {
+                target: { fieldRef: 'field2' },
+                id: '2',
+                operator: FilterOperator.GREATER_THAN_OR_EQUAL,
+                values: ['5'],
+            },
+        ];
+        const expected: DbtColumnLightdashMetric['filters'] = [
+            { field1: '> 4' },
+            { field2: '>= 5' },
+        ];
+        expect(convertMetricFilterToDbt(filters)).toEqual(expected);
+    });
+
+    it('should convert IN_THE_NEXT and IN_THE_PAST filters correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'field1' },
+                id: '1',
+                operator: FilterOperator.IN_THE_NEXT,
+                values: ['14 days'],
```

Comment:
The unit of time is in the `settings`:
  eg:
  ```
              {
                "id": "08d5781b-203a-43b9-a34a-01e14016699c",
                "operator": "inThePast",
                "values": [
                    1
                ],
                "settings": {
                    "unitOfTime": "years",
                    "completed": false
                },
                "target": {
                    "fieldRef": "events.date"
                }
            },
```
Should we return error when `completed: true` since our YML syntax doesn't support it?

Reply:
or add +1 to the value ..